flash_update/admin.py

```diff
@@ -10,7 +10,9 @@
     FlashActionsTaken,
     FlashCountryDistrict,
     FlashEmailSubscriptions,
-    Donors
+    Donors,
+    DonorGroup,
+    FlashUpdateShare
```

Comment:
Let's always add `,` for the last item in the list as well. This creates a better diff for future changes.